<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KnowledgeBase</class>
 <widget class="QWidget" name="KnowledgeBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>572</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>572</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>572</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1080</width>
    <height>572</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Просмотр базы знаний</string>
  </property>
  <widget class="QComboBox" name="alloyComboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="propertyList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>511</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="propertyValuesList">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>490</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Просмотр базы знаний</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
